### PR TITLE
CI: Test against Ruby 2.5.0

### PR DESCRIPTION
**Pull request by jekyllbot**
- [x] Test against Ruby 2.5.0
- [x] Stop testing against Ruby 2.1 (EOL)

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -5,10 +5,11 @@ language: ruby
...
 
 rvm:
-  - &ruby1 2.4.2
-  - &ruby2 2.3.5
-  - &ruby3 2.2.8
-  - &ruby4 2.1.10
+  - &ruby1 2.5.0
+  - &ruby2 2.4.2
+  - &ruby4 2.3.5
+  - &ruby4 2.2.8
```
Comment (ashmaroli) [inline .travis.yml]: duplicate assignment:
```
&ruby4 2.3.5
&ruby4 2.2.8
```

Comment (DirtyF) [reply .travis.yml]: fixed

File: .travis.yml [hunk 2]
```diff
@@ -5,16 +5,17 @@ language: ruby
...
+  - &ruby3 2.3
+  - &ruby4 2.2
+  - &ruby5 2.1
   - &jruby jruby-9.1.14.0
 
 matrix:
   include:
     - rvm: *ruby1
-      env: TEST_SUITE=test ROUGE=1.11.1
+      env: TEST_SUITE=test
```
Comment (parkr) [inline .travis.yml]: Just remove this section entirely including rvm above.

**Comment by DirtyF** [comment]
Travis is green again 💚

**Comment by parkr** [comment]
@DirtyF out of curiosity, why did you remove the more specific versions? Using the most up-to-date versions seems like a good idea?

**Comment by DirtyF** [comment]
@parkr I don't remember CI failing because of an updated patched version of Ruby, added the specificity back with latest versions, travis is still green 💚

**Comment by parkr** [comment]
I guess we also need to remove 2.1 on AppVeyor?

**Comment by parkr** [comment]
Or we set jekyll-watch to 1.x when Ruby 2.1 or below!

**Comment by ashmaroli** [comment]
> Or we set jekyll-watch to 1.x when Ruby 2.1 or below!

We haven't received any complaints of recent releases of `jekyll-watch` breaking an existing workflows.. ergo its safe to assume that Jekyll users still on Ruby 2.1 are fine with having to upgrade their Ruby Version..

So, lets drop testing on Ruby 2.1 its EOL anyways..

**Comment by parkr** [comment]
:+1: to dropping a potentially unsafe version of Ruby. Can happen in another PR.

**Comment by DirtyF** [comment]
#6640 drops support of Ruby 2.1 and 2.2

**Comment by parkr** [comment]
#6640 adds dependency update checks to jekyll doctor. I don’t see a PR which removes 2.1. You can add it here too.

**Comment by ashmaroli** [comment]
> `#6640` drops support of Ruby 2.1 and 2.2

I think you pointed towards the wrong PR.. Either ways, existing PRs propose dropping library support for older Rubies entirely.. I was suggesting just stop *testing* on outdated Ruby versions..

**Comment by DirtyF** [comment]
I meant #6560

**Comment by ashmaroli** [comment]
please drop 2.1 on AppVeyor too..

**Comment by parkr** [comment]
Feel free to merge once Travis & AppVeyor are both :green_heart:

**Comment by DirtyF** [comment]
@jekyllbot: merge +dev

**Pull request merged**